regexes/client/browsers.yml

```diff
@@ -1722,6 +1739,16 @@
   engine:
     default: 'WebKit'
 
+# iBrowser Mini (https://www.goibibo.com/)
+- regex: 'iBrowser/Mini(\d+\.\d+)'
+  name: 'iBrowser Mini'
+  version: '$1'
+
+# iBrowser (https://www.goibibo.com/)
```

Comment:
https://www.goibibo.com/
it looks like this site no longer applies to this browser, on the flight ticket sales page
I suggest removing the link to the site.